src/api/render-node.ts

```diff
@@ -74,16 +73,15 @@ export function renderNodeManagement(snapshot: StateSnapshot, config: ServiceCon
 
 // helpers
 
-function getSigner(snapshot: StateSnapshot, config: ServiceConfiguration) {
+function getSigner(snapshot: StateSnapshot) {
   const version = snapshot.CurrentImageVersions['main']['signer'];
   if (!version) return undefined;
 
   return {
     InternalPort: 7777,
     Disabled: false,
     DockerConfig: {
-      Image: `${config.DockerNamespace}/signer`,
-      Tag: version,
+      ...parseImageTag(version),
```

Comment:
parseImageTag is dangerous, it may not return valid response:

For example, what happens if somebody has a typo while changing the JSON file and they input an invalid image name?

1. parseImageTag can throw an exception which would kill the entire render flow and cause Boyar to bootloop management service

2. parseImageTag can return an empty "Image" - what does that do to Boyar? Seems like something Boyar won't be able to handle

The error flows should be just like lines 77-78, if for some reason we don't get a valid version that can be split into a valid Image and Tag, we should return undefined for the entire service, meaning still render the API response for Boyar, just the service should be missing.